### PR TITLE
Update README.md to use Homebrew-versions

### PR DESCRIPTION
**Pull request by dalinaum**
@zaki50's android-ndk-r10e should be merged into Homebrew-versions before this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,17 +61,18 @@ Prerequisites:
...
+    brew tap homebrew/versions
+    brew install android-ndk-r10e
     ```
 
  * Add two environment variables to your profile:
 
     ```
     export ANDROID_HOME=~/Library/Android/sdk
-    export NDK_HOME=/usr/local/Cellar/android-ndk/r10e
+    export NDK_HOME=NDK_HOME=/usr/local/Cellar/android-ndk/r10e
```
Comment (zaki50) [inline README.md]: remove `NDK_HOME=`

**Comment by zaki50** [comment]
:+1:

**Comment by zaki50** [comment]
I created the PR for homebrew-versions https://github.com/Homebrew/homebrew-versions/pull/1196

**Comment by stk1m1** [comment]
:+1: I'd suggest this be merged after @zaki50 's PR gets accepted there.

**Comment by zaki50** [comment]
@dalinaum Homebrew/homebrew-versions#1196 has been merged. please merge this PR.

**Comment by dalinaum** [comment]
Unfortunately, CI server doesn't work now. We will wait until CI will be reboot.

**Comment by dalinaum** [comment]
retest this please.

**Comment by dalinaum** [comment]
retest this please

**Comment by dalinaum** [comment]
retest this please

**Pull request merged**